Changes

```diff
@@ -156,6 +156,9 @@ Working version
 - #13740: Improve performance of Weak.find_aux
   (Josh Berdine, review by Gabriel Scherer)
 
+- #15049: Do not use Obj.Extension_constructor.of_val in Type.Id
```

Comment:
Nit, but Changes entries should be written from the point of view of the reader, not the writer. Here, a random reader who comes across this entry will not know how or why this would be relevant to her. Instead, something like `Avoid use of Obj.Extension_constructor.of_val in Type.Id for better performance.` or `Type.Id: replace Obj.Extension_constructor.of_val by [%extension_constructor] for better performance.` (which has the advantage of publicizing `[%extension_constructor]`) would be better in this respect.

Reply:
Looks great, thanks!